moveit_core/CMakeLists.txt

```diff
@@ -143,6 +143,8 @@ catkin_package(
     OCTOMAP
     urdfdom
     urdfdom_headers
+  CFG_EXTRAS
+    Coverage.cmake
```

Comment:
Is this line sufficient to include Coverage.cmake in all downstream packages?

Reply:
Yes.